pkgs/by-name/ca/catalyst/package.nix

```diff
@@ -0,0 +1,85 @@
+{ stdenv
+, lib
+, fetchurl
+, appimageTools
+, makeWrapper
+, electron_30
+, electronPackage ? electron_30
+, asar
+}:
+
+let
+  electron = electronPackage;
+in
+stdenv.mkDerivation rec {
+  pname = "catalyst";
+  version = "3.8.5";
+
+  src = fetchurl {
+    url = "https://github.com/CatalystDevOrg/Catalyst/releases/download/v${version}/catalyst-${version}.AppImage";
+    hash = "sha256-L9jrtr3T1PgZRVds3y7SNHBPVv81uJpSfCpxSAJiwwM=";
+    name = "${pname}-${version}.AppImage";
+  };
+
+  buildInputs = [
+    asar
+  ];
+
+  appimageContents = appimageTools.extractType2 {
+    name = "${pname}-${version}";
+    inherit src;
+  };
+
+  dontUnpack = true;
+  dontConfigure = true;
+  dontBuild = true;
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin $out/share/${pname} $out/share/applications
+    mkdir -p $out/share/${pname}/resources/
+
+    cp -a ${appimageContents}/locales $out/share/${pname}
+    cp -a ${appimageContents}/catalyst.desktop $out/share/applications/${pname}.desktop
+    mkdir -p $out/share/pixmaps
+    cp -r ${appimageContents}/usr/share/icons/hicolor/0x0/apps/catalyst.png $out/share/pixmaps/
+    asar extract ${appimageContents}/resources/app.asar resources/
+    rm -rf resources/.github
+    rm -rf resources/.vscode
+    rm -rf resources/.eslintrc.json
+    rm -rf resources/.gitignore
+    rm -rf resources/.pnpm-debug.log
+    rm -rf resources/contributing.md
+    rm -rf resources/pnpm-lock.yaml
+    rm -rf resources/README.md
+    rm -rf resources/CODE_OF_CONDUCT.md
+    rm -rf *.nix
+    substituteInPlace resources/src/index.html \
+      --replace-fail 'catalyst-default-distrib' 'catalyst-default-nixpkgs'
+
+    substituteInPlace $out/share/applications/${pname}.desktop \
+      --replace-fail 'Exec=AppRun' 'Exec=${pname}'
+
+    asar pack resources/ $out/share/${pname}/resources/app.asar
+
+    runHook postInstall
+  '';
+
+  postFixup = ''
+    makeWrapper ${electron}/bin/electron $out/bin/${pname} \
+      --add-flags $out/share/${pname}/resources/app.asar \
+      --prefix LD_LIBRARY_PATH : "${lib.makeLibraryPath [ stdenv.cc.cc ]}"
+  '';
+
+  meta = with lib; {
+    description = "A minimal, functional, and customizable user-focused FOSS web browser ";
+    homepage = "https://getcatalyst.eu.org";
+    license = licenses.mit;
```

Comment:
```suggestion
  meta = {
    description = "Minimal, functional, and customizable user-focused FOSS web browser based on QtWebEngine";
    homepage = "https://getcatalyst.eu.org";
    license = lib.licenses.mit;
```

Reply:
The Catalyst 5 Alpha uses QtWebEngine, this is Catalyst 3. I will make the other requested changes.

Reply:
Ah I see. In that case a more precise homepage value may also be good.